bin/_tag.sh

```diff
@@ -21,11 +21,12 @@ named_tag() {
 }
 
 head_root_tag() {
-    if clean_head ; then
-        clean_head_root_tag
-    else
-        echo "dev-$(git_sha_head)-$USER"
-    fi
+    echo "stable-2.4.0"
```

Comment:
Superfluous double quotes. No quotes needed at all here.

Reply:
Hi @joakimr-axis

Thank you for your time reading this PR, but this PR is just a PoC to make Linkerd work in ARM arch. I don't have intention to merge it.